using-mobile-extensions/adobe-campaignclassic/README.md

```diff
@@ -0,0 +1,245 @@
+# Adobe Campaign Classic
+
+## Configure Campaign Classic Extension in Launch
+
+1. In Launch click the **Extensions** tab.
+2. On the **Catalog** tab, locate the **Adobe Campaign Classic** extension and click **Install**.
+3. Provide extension settings \(see screen capture below\)
+4. Click **Save**.
+5. Follow the publishing process to update SDK configuration
+
+### Configure Campaign Classic Extension
+
+![ACC-configure-extension](../../.gitbook/assets/ACC-configure-extension.png)
+
+{% hint style="info" %}
+Trying to find your ACC registration or tracking endpoint URLs? Contact your program manager.
+{% endhint %}
+
+#### Registration Endpoints
+
+Provide registration endpoint URL\(s\) for your Campaign Classic instances. You may specify up to three unique endpoints for your development, staging, and production environments. 
+
+#### Tracking Endpoints
+
+Provide tracking endpoint URL\(s\) for your Campaign Classic instances. Like the registration URL's, you may specify up to three unique endpoints for your development, staging, and production environments. 
+
+#### Integration Key (iOS)
+
+Specify up to three unique iOS integration keys for your development, staging, and production environments. iOS integration keys are generated after creating a service containing iOS applications using the Adobe Campaign Classic [client console](https://docs.campaign.adobe.com/doc/AC/en/INS_Installation_steps_for_Windows__Installing_the_client_console.html). For more information on where to find the integration key, please see [Configuring the mobile application in Adobe Campaign](https://docs.campaign.adobe.com/doc/AC/en/DLV_Sending_push_notifications_Setting_up_mobile_app_channel.html#Configuring_the_mobile_application_in_Adobe_Campaign).
```

Comment:
please change the first sentence to start:

"You can specify up to three unique..."